### ISSUE TITLE
Cant start

### ISSUE THREAD

**Issue by AlexBerington** [p2pool/p2pool#317]
Error getting work from bitcoind:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/twisted/internet/defer.py", line 588, in _runCallbacks
    current.result = callback(current.result, _args, *_kw)
  File "/usr/lib/python2.7/dist-packages/twisted/internet/defer.py", line 1184, in gotResult
    _inlineCallbacks(r, g, deferred)
  File "/usr/lib/python2.7/dist-packages/twisted/internet/defer.py", line 1126, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "/usr/lib/python2.7/dist-packages/twisted/python/failure.py", line 389, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
--- <exception caught here> ---
  File "/home/alex/p2pool/p2pool/util/deferral.py", line 41, in f
    result = yield func(_args, *_kwargs)
  File "/usr/lib/python2.7/dist-packages/twisted/internet/defer.py", line 1126, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "/usr/lib/python2.7/dist-packages/twisted/python/failure.py", line 389, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File "/home/alex/p2pool/p2pool/bitcoin/helper.py", line 53, in getwork
    work = yield go()
  File "/usr/lib/python2.7/dist-packages/twisted/internet/defer.py", line 1126, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "/usr/lib/python2.7/dist-packages/twisted/python/failure.py", line 389, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File "/home/alex/p2pool/p2pool/util/jsonrpc.py", line 133, in _http_do
    raise Error_for_code(resp['error']['code'])(resp['error']['message'], resp['error'].get('data', None))
p2pool.util.jsonrpc.NarrowError: -10 Litecoin is downloading blocks...

**Comment by p3yot3** [comment]
The clue is in the message:

 p2pool.util.jsonrpc.NarrowError: -10 **Litecoin is downloading blocks...**

Wait for your wallet to sync before starting p2pool.

**Comment by zhongchaojian** [comment]
p3yot3  whats your Email address. How many peonycredit do you have, I will get them all
